src/createServer.js

```diff
@@ -1,3 +1,45 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const getErrors = require('./utils/getErrors');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    let errors = [];
+
+    try {
+      res.setHeader('Content-Type', 'application/json');
+
+      const splittedUrl = req.url.split('?');
+      const originalText = splittedUrl[0].slice(1);
+      const targetCase = new URLSearchParams(splittedUrl[1]).get('toCase');
+
+      errors = getErrors(originalText, targetCase);
+
+      if (errors.length) {
+        throw new Error();
+      }
```

Comment:
You can throw errors in `getErrors` function if any so you don't need to do it here manually